### ISSUE TITLE
Page specific embed token

### ISSUE THREAD

**Issue by kksevta** [Microsoft/PowerBI-JavaScript#211]
Hi
We can specify page name in embed configuration Details.
But I am curious to know If we can create embed token for one specific page of a report.

**Comment by ali-hamud** [comment]
We don't have this feature yet.
can you please give a scenario that this is necessary?
you can always create multiple reports.

**Comment by kksevta** [comment]
**In case of Non Power BI users embedding**.
If I want to restrict one particular page of a report for a user,  I think there should be a way to create embed token for one particular page.

**Comment by ali-hamud** [comment]
Sorry, my question was:
What additional functionality will this give you? You can reach the same by creating multiple reports.

**Comment by kksevta** [comment]
@ali-hamud Yeah it is correct that I can reach the same by creating multiple reports.
But I am assuming it will be easy to manage one report for multiple users and show different pages on basis of different user roles.
Kindly let me know your thoughts on this.
